manager-dashboard/.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+- Addresses XXX
```

Comment:
a general question: would we use the existing `.github/` folder in the repo instead of this one?

Reply:
These are for specific components (pulled when we merged another repo to this one). Maybe we can discuss further on this.